gameengine.js

```diff
@@ -34,6 +35,14 @@ class GameEngine
         this.surfaceWidth = this.ctx.canvas.width;
         this.surfaceHeight = this.ctx.canvas.height;
         this.timer = new Timer();
+        this.blink = blink;
```

Comment:
So here is where you can do the loop and assign it to the camera instantiation below